ENG4000-Team-A/capstone-project
# #83 Testing React Code Commit

Documentation:
https://reactjs.org/docs/testing.html
https://testing-library.com/docs/react-testing-library/api/

Resource Used:
https://www.youtube.com/watch?v=3e1GHCA3GP0
https://testing-library.com/docs/react-testing-library/example-intro/

Requirement:
Do npm install when in the frontend directory
node.js version > 16.x.x.x

How do I Test?
Go to the frontend directory folder 

Then type in your bash “npm test”. What will happen is that npm will check through all the folders and look for files that have a “test” suffix. 

Should look like this as below
![image](https://user-images.githubusercontent.com/60308758/157803139-08267b1d-f03b-4fcf-8d1b-a3cdca7b617b.png)

Currently have a bit of a router problem with setting up a testing suite that is using the router function. This task is long overdue, so I will be submitting on what I have done. Will need to be a new task if we need to add more unit tests.
